packages/scss/src/commons/vars.scss

```diff
@@ -63,14 +63,8 @@
 	--commons-background-base: var(--palettes-neutral-25);
 	--commons-divider-color: var(--palettes-neutral-200);
 
-	@each $breakpoint, $value in config.$breakpoints {
-		--commons-container-maxWidth#{$breakpoint}: #{function.pxToRem($value)};
```

Comment:
I'm not sure it's a good idea to move theses variables, as they are used in Empty State, Footer and Horizontal Navigation components